cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,27 @@
 /// <reference types="cypress" />
 
-describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+describe('Site page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io');
+  });
+
+  cy.get('h1').should('contain.text', 'Sign In');
+
+  cy.contains('Sign in').click();
+
+  cy.get('h1').should('contain.text', 'Sign In');
+
+  it('should fill in email and password fields with valid data', () => {
+    cy.get('[placeholder=Email]')
+      .type('oleksadowbush@gmail.com');
+
+    cy.get('[placeholder=Password]')
+      .type('Oleksa19*');
+
+    it('should click on [Sign in] button ', () => {
+      cy.get('.btn').click();
+    });
 
+    cy.get('nav').contains('OleksaD19').should('exist');
```

Comment:
The assertion 'cy.get('nav').contains('OleksaD19').should('exist');' should be placed inside a test case (it block).